src/components/MyKiva/GoalSetting/GoalSelector.vue

```diff
@@ -161,9 +165,13 @@ const titleText = computed(() => {
 });
 
 const subtitleText = computed(() => {
+	let extraText = '';
+	if (womenLoansThisYear.value > womenLoansLastYear.value) {
+		extraText = `<br> You've already made ${womenLoansThisYear.value}.`;
```

Comment:
Do we have a mock of this? If not, I think using a single line of text, as shown in the ticket might make sense. It would also allow you to remove the v-html usage.